ios/Plugin/FirebaseAuthentication.swift

```diff
@@ -7,6 +7,8 @@ public typealias AuthStateChangedObserver = () -> Void
 
 @objc public class FirebaseAuthentication: NSObject {
     public let errorDeviceUnsupported = "Device is not supported. At least iOS 13 is required."
+    public let errorInvalidCustomToken = "Invalid custom token."
+    public let errorCustomTokenUnimplemented = "signInWithCustomToken only works on the native layer. Use the js-sdk directly for the web layer https://firebase.google.com/docs/auth/web/custom-auth."
```

Comment:
```suggestion
    public let errorCustomTokenSkipNativeAuth = "signInWithCustomToken cannot be used in combination with skipNativeAuth.";
```

I suggest that we keep the error message simple here and add a helpful description for it in the `definitions.ts` file.